numismatic/feeds/bravenewcoin.py

```diff
@@ -1,9 +1,13 @@
-from .base import Feed
+import logging
 
+from .base import Feed, RestApi
 from ..libs.config import get_config
 
 
-class BraveNewCoinFeed(Feed):
+logger = logging.getLogger(__name__)
```

Comment:
Can this logging be "DRY'd" by moving it to some base class. Essentially, when I make a new entity, I would like it so that as much infrastructure as possible is handled for me. All I essentially would need to do is to make the calls etc.

Reply:
Yeah we could. It's common to have this at the module level and given it's only two lines I don't think it's too bad but we could add it to the Feed base class.

Reply:
Let's not rely upon convention, specially if we want people to add things on. We should try provide as much infrastructure as possible